src/client/App.js

```diff
@@ -13,6 +13,7 @@ import Menu from './components/Menu/Menu.component';
 import LogInPage from './containers/LogInPage/LogInPage.Container';
 import Footer from './components/Footer/Footer.component';
 import './hooks/useProducts';
+import SpecialOfferPage from './containers/SpecialOfferPage/SpecialOfferPage.Container';
```

Comment:
Also can you change the file name to SpecialOfferPage.container.js

Reply:
Changes Updated Thankyou.